src/applications/caregivers/tests/e2e/1010cg-facilities-api.cypress.spec.js

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+import featureTogglesFacilitiesApi from './fixtures/mocks/feature-toggles-facilities-api.json';
+import formConfig from '../../config/form';
+import manifest from '../../manifest.json';
+
+import { setupPerTest, pageHooks } from './utils/helpers';
+
+export const setupPerTestWithFacilitiesFeatureToggle = () => {
+  setupPerTest();
+  cy.intercept('GET', '/v0/feature_toggles?*', featureTogglesFacilitiesApi);
```

Comment:
Using all of the normal default intercepts except for the feature toggles to get the toggles with it turned on.